flake.nix

```diff
@@ -47,6 +47,13 @@
             (import ./editors/nvim-lsp.nix { inherit pkgs; })
           ];
         };
+        devShells.vscodium = pkgs.mkShell {
+          nativeBuildInputs = [
+            nixd
+            pkgs.nixpkgs-fmt
```

Comment:
Maybe we can share the same dependency expression with `devShells.nvim`, and if the dependency is changed in the future, it can be updated it consistently